src/Destructurama.Attributed/Destructurama.Attributed.csproj

```diff
@@ -4,12 +4,23 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>Use attributes to control how complex types are logged to Serilog.</Description>
     <RootNamespace>Destructurama</RootNamespace>
+    <IsPackable>true</IsPackable>
     <PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
+    <PackageId>Destructurama.Attributed</PackageId>
+    <PackageReleaseNotes>
+      <![CDATA[
+v1.0
+- Migrated from Serilog.Extras.Attributed at serilog/serilog.
+]]>
+    </PackageReleaseNotes>
```

Comment:
no, no, please, let's not clog configuration

Reply:
@sungam3r How would you like to solve your release notes?

Reply:
All release notes are in releases: https://github.com/destructurama/attributed/releases

They are generated/formatted automatically via GitHub when I publish release. Simply speaking it is not the best solution to put the history of releases inside the package itself (at least when you use GitHub releases expirience).

Reply:
Do you know if it ignores the `<ReleaseNotes>` tag completealy? In that case, just adding a single statement there works perfectly as explanation on how it is solved. It should also be added to our documentation obviously.

Reply:
What "it"? GitHub releases feature?

Reply:
yep. SO bu just having ` <PackageReleaseNotes>Handled by GitHub release</PackageReleaseNotes>` you should be fine.

Reply:
Github release experience knows nothing about nuget/project structure.

> <PackageReleaseNotes>Handled by GitHub release

It is nothing more that stub just to satisfy opionated rule, but OK, in such form it may have value, a little.